cdap-master-spi/src/main/java/io/cdap/cdap/master/spi/twill/SecurityContext.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.master.spi.twill;
+
+/**
+ * Specifies security context of a {@link org.apache.twill.api.TwillRunnable}.
+ * This includes its identity, along with userId and groupId.
+ */
+public class SecurityContext {
+
+  private final Long userId;
+  private final Long groupId;
+  private final String identity;
+
+  private SecurityContext(SecurityContextBuilder builder) {
+    if (builder.identity != null) {
+      this.identity = builder.identity;
+    } else {
```

Comment:
usually there is no need to do else part.
https://github.com/cdapio/cdap/blob/83798c0cc09cb616a54d1065962e07c2d04c21ce/cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/spec/StageSpec.java#L56